harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -569,7 +577,8 @@ def _compute_validation_metrics(self) -> workload.Response:
                 )
 
                 metrics = self._convert_metrics_to_numpy(metrics)
-                num_inputs = self.context.get_per_slot_batch_size() * len(self.validation_loader)
+                num_batches = len(self.validation_loader)
+                num_inputs = self.context.get_per_slot_batch_size() * num_batches
```

Comment:
this seems like a strong assumption that the dataloader used for evaluate_full_dataset returns the per slot batch.  what do you think about creating an iterator over dataloader to get a batch and use that for batch size?

Reply:
what would you say about just _not_ logging this data for `evaluate_full_dataset`?

I'm thinking this is an infrequently used feature, and the only real reason to use it is if you are doing something pretty custom (multiple passes through the dataset or something?).  I'm also thinking they can basically gather the same metric on their own in this case.

Reply:
excluding this for `evaluate_full_dataset` make sense to me.